src/client/Microsoft.Identity.Client/ManagedIdentity/ServiceFabricManagedIdentitySource.cs

```diff
@@ -45,8 +45,42 @@ public static AbstractManagedIdentity Create(RequestContext requestContext)
             return new ServiceFabricManagedIdentitySource(requestContext, endpointUri, EnvironmentVariables.IdentityHeader);
         }
 
+        internal override Func<HttpClient> ValidateServerCertificateCallback(RequestContext requestContext)
```

Comment:
better name might be "CustomHttpClientCreator" but maybe you don't need to use a function pointer / Func here at all?